frontend/templates/product-list/sections/FilterableProductsSection/useProductSearchHitPricing.ts

```diff
@@ -29,16 +31,31 @@ export function useProductSearchHitPricing(
 
    const price = proTierPrice ?? product.price_float;
```

Comment:
I think this is causing the pro price to always be preferred, even if it's greater than the price that is available to all other users.